requires langinfo.h support not present in musl, so use the embedded
localisation facilities instead
diff --git a/build/cmake/setup.cmake b/build/cmake/setup.cmake
index 365c1f3..89a1610 100644
--- a/build/cmake/setup.cmake
+++ b/build/cmake/setup.cmake
@@ -604,7 +604,7 @@ if(NOT MSVC_VERSION LESS 1600)
     check_include_file_cxx(type_traits HAVE_TYPE_TRAITS)
 endif()
 check_include_file(fcntl.h HAVE_FCNTL_H)
-check_include_file(langinfo.h HAVE_LANGINFO_H)
+#check_include_file(langinfo.h HAVE_LANGINFO_H)
 check_include_file(sched.h HAVE_SCHED_H)
 check_include_file(unistd.h HAVE_UNISTD_H)
 check_include_file(wchar.h HAVE_WCHAR_H)
diff --git a/build/cmake/setup.h.in b/build/cmake/setup.h.in
index bce33a7..9ba4d76 100644
--- a/build/cmake/setup.h.in
+++ b/build/cmake/setup.h.in
@@ -1183,7 +1183,6 @@
 #cmakedefine ICONV_CONST @ICONV_CONST@
 
 /* Define if you have the <langinfo.h> header file.  */
-#cmakedefine HAVE_LANGINFO_H 1
 
 /* Define if you have the <sys/soundcard.h> header file. */
 #cmakedefine HAVE_SYS_SOUNDCARD_H 1
